Add Lightning Talks Blog Post (#263)
SeaGL/seagl.github.io: _posts/2021-08-18-Lightning-Talks.md

---
layout: post

> title: 'Lightning Talks at SeaGL!'

Reviewer: We've been saying "SeaGL 2021" in titles related to this year's event, so it might be worth changing that for consistency's sake. Not a big deal, though.

Author: ```suggestion
title: 'Lightning Talks at SeaGL 2021!'
```

status: publish
type: post
published: true
categories: news
---

This year we’re incredibly excited to announce that we’re hosting lightning talks at SeaGL!
Think of this space as a great opportunity to learn about fun topics,

> or even present your own talk (maybe even for the first time!). 

Reviewer: Maybe add something about using it as a way to explore the kernel of a future talk idea. Perhaps change the paragraph to

> We’re incredibly excited to announce that we’re hosting lightning talks at SeaGL this year!
> Think of this space as a great opportunity to learn about fun topics, or even present your own talk.
> It's a great way to get comfortable with speaking to an audience or to explore the kernel of an idea that could become a full-length talk at SeaGL 2022!

> This year we’re incredibly excited to announce that we’re hosting lightning talks at SeaGL!
> Think of this space as a great opportunity to learn about fun topics,
> or even present your own talk (maybe even for the first time!). 

Author: ```suggestion
We’re incredibly excited to announce that we’re hosting lightning talks at SeaGL this year!
Think of this space as a great opportunity to learn about fun topics, or even present your own talk.
It's a great way to get comfortable with speaking to an audience or to explore the kernel of an idea that could become a full-length talk at SeaGL 2022!
```

The lightning talks will be a series of 5 minute talks which will be pre-recorded - meaning
that we can accommodate speakers from all over the world!
We’re looking for any community focused talks.
A few examples are:

* Tech examples: Event organization, coding, design, documentation, open source, etc. 
* Non-tech examples: non-profit work, hobbies/programs (eg. community gardens), sports teams, theater, etc.

> Our one ask is that there are no promotions of any kind - meaning, please no video resumes, no company/product promos, etc. 

Reviewer: "'Ask' is a verb!" he shouts into the void. :-)

Author: ```suggestion
Our one request is that there are no promotions of any kind - meaning, please no video resumes, no company/product promos, etc. 
```

> We’re looking forward to hearing from you, our community. Please submit your talk ideas [here](https://forms.gle/3iVpzavUxmjLTXmm8) **before September 8, 2021**.

Reviewer: Maybe instead

> Please [submit your talk ideas](https://forms.gle/3iVpzavUxmjLTXmm8)

"Here" as link wording is dead weight. Alternatively, you could replace "here" with "in the Google form".

Author: ```suggestion
We’re looking forward to hearing from you, our community. Please [submit your talk ideas](https://forms.gle/3iVpzavUxmjLTXmm8) **before September 8, 2021**.
```
